app/Http/Controllers/Admin/AdminUserController.php

```diff
@@ -36,4 +43,91 @@ public function list(): JsonResource
     {
         return AdminUserResource::collection(User::query()->with(['members'])->get());
     }
+
+    #[OA\Post(
+        path: '/api/v1/admin/user',
+        description: 'Permissions: ' . Permission::globalAdmin->name,
+        summary: 'Create user',
+        requestBody: new OA\RequestBody(
+            content: new OA\JsonContent(
+                required: ['name', 'hasGlobalAdmin'],
+                properties: [
+                    new OA\Property(property: 'name', type: 'string', example: 'Jan'),
+                    new OA\Property(property: 'email', type: 'string', example: 'jan@jan.pl'),
```

Comment:
niektóre z tych rzeczy są nullable, możesz przejrzeć to i patch

Reply:
w sensie o co dokładnie pytasz? rzeczy nie nullable są zaznaczone w dokumentacji jako required